dbms/src/Flash/tests/gtest_compute_server.cpp

```diff
@@ -286,5 +291,50 @@ try
 }
 CATCH
 
+TEST_F(ComputeServerRunner, cancelAggTasks)
```

Comment:
Note that cancel tests must be called after mppTasks tests because of the sleep may cause mppTasks to fail